CODEOWNERS

```diff
@@ -40,6 +40,7 @@ Makefile*                                                        @istio/wg-test-
 /pkg/webhooks/                                                   @istio/wg-environments-maintainers
 /prow/                                                           @istio/wg-test-and-release-maintainers
 /release/                                                        @istio/wg-test-and-release-maintainers
+/releasenotes/                                                   @istio/wg-test-and-release-maintainers
```

Comment:
did we decide to add release mgr here as well?

Reply:
Added. Thanks :)